src/io/flutter/logging/FlutterLogView.java

```diff
@@ -441,11 +468,30 @@ private Object getTagForPosition(MouseEvent e) {
     fixColumnWidth(logTree.getColumn(CATEGORY), 110);
     logTree.getColumn(MESSAGE).setMinWidth(100);
 
+    setupLogTreeScrollPane();
+  }
+
+  private void setupLogTreeScrollPane() {
     final JScrollPane pane = ScrollPaneFactory.createScrollPane(
       logTree,
       ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED,
       ScrollPaneConstants.HORIZONTAL_SCROLLBAR_AS_NEEDED);
+    pane.getVerticalScrollBar().getModel().addChangeListener(new LogVerticalScrollChangeListener() {
+      @Override
```

Comment:
It would be great if the ScrollToEnd toggle action could have its state updated as well.  I believe it will be out of sync?

Reply:
Ah, I moved that logic to ScrollToEndAction. so it is more clear.
Please check my record in the description of this PR.

Reply:
Fantastic.  Thanks!